input/europe/template.xml.in

```diff
@@ -0,0 +1,28 @@
+ <facility>
+    <!-- {{ country }} -->
+    <name>{{ reactor_name }}</name>
+    <config>
+      <Reactor>
+
+        <fuel_inrecipes>  <val>uox_fuel_recipe</val>      <val>mox_fuel_recipe</val>      </fuel_inrecipes>
+        <fuel_outrecipes> <val>uox_used_fuel_recipe</val> <val>mox_used_fuel_recipe</val> </fuel_outrecipes>
+        <fuel_incommods>  <val>uox</val>                  <val>mox</val>                  </fuel_incommods>
+        <fuel_outcommods> <val>waste</val>                <val>waste</val>                </fuel_outcommods>
+        <fuel_prefs>      <val>0.1</val>                  <val>1.0</val>                  </fuel_prefs>
+
+        <cycle_time>18</cycle_time>
+        <refuel_time>2</refuel_time>
+        <assem_size>523.4</assem_size>
+        <n_assem_core>{{ n_assem_core}}</n_assem_core>
+        <n_assem_batch>{{n_assem_batch}}</n_assem_batch>
+
+        <pref_change_times>   <val>4</val>   </pref_change_times>
+        <pref_change_commods> <val>uox</val> </pref_change_commods>
+        <pref_change_values>  <val>2.0</val> </pref_change_values>
+        <power_cap>{{capacity}}</power_cap>
+
+      </Reactor>
+    </config>
+  </facility>
+
+
```

Comment:
may want to be careful about extra newlines. These are allowed in xml in most places, but the final file will be easier to handle without them.